docs/cel2/decision-tree.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Cel2 Decision Tree
+description: Not sure what you need to do to migrate from Celo L1 to Celo L2?
+---
+
+<details>
+  <summary>I run a node</summary>
+  
+  Follow the [node upgrade instructions](docs/cel2/l2-operator-guide.md).
+
+</details>
+<details>
+  <summary>I use or need a node provider</summary>
+
+  The following node providers will be available shortly after the migration to Cel2:
+
+  - [Quicknode](https://www.quicknode.com/chains/celo) (not yet verified)
+  - [Infura](https://www.infura.io/networks/celo) (not yet verified)
+  - [DSRV](https://drpc.org/chainlist/celo) (not yet verified)
+
+
+  If none of the above work for you, you can use cLabs best effort hosted node: [Forno](https://docs.celo.org/network/node/forno).
+</details>
+<details>
+  <summary>I use ContractKit or Fee Abstraction (ERC-20 gas tokens)</summary>
+
+  Recommended versions for SDK and libraries:
```

Comment:
If it's required, I would change it to "required"

Reply:
It's not, strictly speaking. The required versions are much lower, but @aaronmboyd made a good point in not encouraging people to use old versions at all and always point them to the latest.

Reply:
That might have been @aaronmgdr hehe

Reply:
True - Aarons everywhere 😆